### PR TITLE
feat(repulsion effect): Add example of Repulsion Effect 

### PR DESCRIPTION
**Pull request by damienmontastier**
This is a example inspired by the [tutorial of repulsion effect](https://tympanus.net/codrops/2018/12/06/interactive-repulsion-effect-with-three-js/) from [Codrops](https://codrops.com/) adapted for Tresjs

### CODE REVIEW

File: components/content/repulsion-effect/index.vue [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  alpha: true,
+  shadows: true,
+  shadowMapType: PCFSoftShadowMap,
+  powerPreference: "high-performance",
+}
+
+
+watchEffect(() => {
+  if (rectAreaLightRef.value) {
+    rectAreaLightRef.value.lookAt(0, 0, 0);
```
Comment (JaimeTorrealba) [inline components/content/repulsion-effect/index.vue:18]: Just as a question, if you add the lookAt at the component it doesn't work?

Like <TresRectAreaLight ref="rectAreaLightRef" color="#341212" :decay="0" :width="1000" :height="1000"
      :position="[5, 20, 50]" :look-at="[0,0,0]" />

File: components/content/repulsion-effect/index.vue [hunk 2]
```diff
@@ -0,0 +1,133 @@
+<script setup lang="ts">
+import { TresCanvas, useTresContext } from '@tresjs/core'
+import { PCFSoftShadowMap, BasicShadowMap, PCFShadowMap, NoColorSpace, LinearSRGBColorSpace, SRGBColorSpace, NoToneMapping } from 'three'
+import { OrbitControls, Box, vAlwaysLookAt, vLightHelper } from '@tresjs/cientos';
```
Comment (JaimeTorrealba) [inline components/content/repulsion-effect/index.vue:4]: Do you need to import them?
I think we have auto-import from nuxt here 

Also, some imports are not been utilized

**Review by JaimeTorrealba — COMMENTED** [review]
Hi my friend this look awesome, I drop some questions and suggestions

The cone and the cylinder are rotating way too fast in comparison with the original demo, could you check that little detail?

Also, please remove the package-lock from the PR

File: components/content/repulsion-effect/index.vue (hunk 1 above)
Comment (damienmontastier) [reply components/content/repulsion-effect/index.vue:18]: You are right, I didn't actually know that you could use the `look-at` like that.

File: components/content/repulsion-effect/index.vue (hunk 2 above)
Comment (damienmontastier) [reply components/content/repulsion-effect/index.vue:4]: Exactly! Small error on my part!

**Comment by damienmontastier** [comment]
Hey @JaimeTorrealba , I've updated everything 😉

**Review by JaimeTorrealba — APPROVED** [review]
I look everything fine. Thanks for the adjustments

PD: Remember that you shouldn't mark conversations as resolve (only the reviewer) :D

**Comment by damienmontastier** [comment]
@JaimeTorrealba My bad, thanks for the advice 😊

**Pull request merged**